src/main/java/jp/ac/keio/bio/fun/xitosbml/CuiRun.java

```diff
@@ -0,0 +1,52 @@
+package jp.ac.keio.bio.fun.xitosbml;
+
+import java.util.concurrent.Callable;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "Running XitoSBML-CUI", mixinStandardHelpOptions = true, description = "Saves SBML document and spatial image", version = "1.0")
+
+public class CuiRun implements Callable<Integer> {
+
+	@Option(names = "-i", required = true, description = "The input option")
+	String iValue;
+	@Option(names = "-o", required = true, description = "The output option")
```

Comment:
Maybe better to specify that the output should be an XML file.  For example, `the path to output XML file`.

Reply:
Please refer to https://github.com/Medha-B/XitoSBML/pull/7